ui/app/components/tx-view.js

```diff
@@ -41,6 +49,16 @@ function TxView () {
 }
 
 TxView.prototype.render = function () {
+
+  var props = this.props
```

Comment:
@lhr0909  - can you familiarize yourself with the motivation behind selectors?
We'd like to avoid copy-pasting L53-57 every time we need to use a balance.

http://redux.js.org/docs/recipes/ComputingDerivedData.html
https://github.com/reactjs/reselect

FYI:
The state management logic is super hacky on my WIP branch, because I'm getting the responsive UI styled - don't use it as an example.

lmk if you have any qs